pkg/cronjob/es_index_cleaner.go

```diff
@@ -10,7 +10,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
```

Comment:
there is no need to do this `v1` is the default.

Reply:
Sorry that was gofmt not me. Will fix